arcgis-rest-api/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for arcgis-to-geojson-utils 10.4
```

Comment:
Yes, it corresponds to [this project](https://www.npmjs.com/package/arcgis-to-geojson-utils).